1차/윤형진_모의고사.py

```diff
@@ -0,0 +1,33 @@
+def solution(answers):
+    students_pattern = [[1, 2, 3, 4, 5], [2, 1, 2, 3, 2, 4, 2, 5], [3, 3, 1, 1, 2, 2, 4, 4, 5, 5]]
+    students_score = []
+    score = 0
+    winner = []
+    max_score = 0
+
+    for i in range(len(students_pattern)):
+        k = 0
+        score = 0
+
+        for j in range(len(answers)):
+
+            if k == len(students_pattern[i]):
+                k = 0
+
+            if answers[j] == students_pattern[i][k]:
+                score = score + 1
+
+            k = k + 1
+
+        students_score.append(score)
```

Comment:
```suggestion
    for student_pattern in students_pattern:
        k = 0
        score = 0

        for answer in answers:

            if k == len(student_pattern):
                k = 0

            if answer == student_pattern[k]:
                score = score + 1

            k = k + 1

        students_score.append(score)
```

- 매번 idx로 접근하기 보다는 for-in 문을 좀 더 효율적으로 사용하면 좋을 것 같아요

Reply:
확실히 더 간결해지네요